.github/scripts/ci-build.sh

```diff
@@ -33,16 +33,11 @@ else
     MOVING=0
 fi
 
-cd $MMTK_JULIA_DIR
-
-MMTK_MOVING=$MOVING make $build_type
-
 cd $JULIA_PATH
-
 # Clean first
 make cleanall
 # Build
 cp $BINDING_PATH/.github/scripts/Make.user $JULIA_PATH/
-MMTK_PLAN=$plan MMTK_BUILD=$build_type make
+MMTK_MOVING=$MOVING MMTK_PLAN=$plan MMTK_BUILD=$build_type make
```

Comment:
I suggest adding a comment here, saying that `MMTK_JULIA_DIR` is defined in `common.sh`, so this will build with the current binding directory (rather than the binding version defined in Julia).